content/christine_belzie/create-a-url-using-slugs.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: "Create a URL Using Slugs"
+Description: "Step-by-Step guide on creating a slug with URL ."
+DatePublished: "2022-01-14"
+Categories:
+  - "web-development"
+Tags:
+  - "World Wide Web"
+  - "Link"
+CatalogContent:
+  - "introduction-to-the-web"
+  - "paths/web-development"
+---
+
+[https://downshiftology.com/recipes/gluten-free-chocolate-chip-cookies/]: https://downshiftology.com/recipes/gluten-free-chocolate-chip-cookies/
+[URL]: https://www.codecademy.com/resources/docs/general/url
+[slugify.online]: https://slugify.online/
+[John Mueller from Google(start at 19:00)]: https://www.youtube.com/live/4FkSZIW6d48?feature=share
+
+[Step 1 of URL slug]: https://github.com/Codecademy/ugc/main/content/christine_belzie/step1-of-slug.png
+[Step 2 of URL slug]: https://github.com/Codecademy/ugc/main/content/christine_belzie/step2-of-slug.png
+[Step 3 of URL slug]: https://github.com/Codecademy/ugc/main/content/christine_belzie/step3-of-slug.png 
+
+## Introduction 
+
+Picture this. You're scrolling through the internet, find a cookie recipe, and click on the "Share" button to send this link to your friend:
+[https://downshiftology.com/recipes/gluten-free-chocolate-chip-cookies/]
+
+You see `gluten-free-chocolate-chip-cookies` after the backslash (`/`). This is called a **slug**. Slugs are a vital part of a [URL] that helps identify a particular page on a website in a user-friendly manner. Essentially, slugs identify the content found on the page. In this article, you will learn the importance of slugs in SEO strategy, how to create them, and best practices.   
```

Comment:
Could you specify here which backslash you're referring to and clarify which part exactly uniquely identifies the page

Reply:
Sure